wifite/args.py

```diff
@@ -470,7 +468,7 @@ def _add_command_args(self, commands):
 
 if __name__ == '__main__':
     from .util.color import Color
-    from config import Configuration
+    from .config import Configuration
```

Comment:
PyCharm marked it as an error, so I thought why not add here a little dot

Reply:
Good catch.

`from config` works in Python2 but crashes in Python3.

```bash
$sudo python3 -m wifite.args
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/wifite2/wifite/args.py", line 473, in <module>
    from config import Configuration
ModuleNotFoundError: No module named 'config'
```

`from .config` works in both Python2 & Python3.